test/settings/output.ts

```diff
@@ -0,0 +1,88 @@
+import { focusChild, focusMain, test, useSpectron } from '../helpers/spectron';
+import { setFormDropdown } from '../helpers/spectron/forms';
+import { sleep } from '../helpers/sleep';
+
+useSpectron();
+
+const AUDIO_BITRATES = [
+  64,
+  96,
+  128,
+  160,
+  192,
+  224,
+  256,
+  288,
+  320,
+  352,
+  384,
+  416,
+  448,
+  480,
+  512,
+  544,
+  576,
+  608,
+  640,
+  672,
+  704,
+  736,
+  768,
+  800,
+  832,
+  864,
+  896,
+  928,
+  960,
+  992,
+  1024,
+];
+
+test('simple settings', async t => {
```

Comment:
this should really be more descriptive but other than that LGTM

Reply:
Changed to "simple output mode settings"